client.go

```diff
@@ -122,10 +124,32 @@ func NewClient(ctx context.Context, p OpenFGAParams) (*Client, error) {
 	}
 	return &Client{
 		api:         api,
-		AuthModelId: p.AuthModelID,
+		authModelId: p.AuthModelID,
+		getStoreID:  client.GetStoreId,
+		setStoreID:  client.SetStoreId,
 	}, nil
 }
 
+// GetAuthModelID gets the currently configured authorization model ID.
```

Comment:
I think I'd do this a bit differently (and drop the non-go-idiomatic "Get" prefix.
```go
func (c *Client) AuthModelID() string {
	return c.authModelID
}

func (c *Client) WithAuthModelID(id string) *Client {
    return &Client{
          api: c.api,
          authModelID: id,
          ...
    }
}
```
So, basically, rather than modifying the id in place, we return a new client for the new id easily. It could be replicated for the store as well.

Reply:
As discussed on a call, dropped the "Get" prefix from the method names, but left the "Set*" methods as is, as the implementation for `WithStoreID` would be very complicated.